api/apps/api/src/modules/cost-surface/cost-surface-module-first.class.ts

```diff
@@ -0,0 +1,8 @@
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { CostSurfaceApiEntity } from './cost-surface.api.entity';
+
+@Module({
+  imports: [TypeOrmModule.forFeature([CostSurfaceApiEntity])],
+})
+export class CostSurfaceModuleFirstClass {}
```

Comment:
Yes, it is totally a tmp name, until, after sync-ing with @KevSanchez, we move everything related to cost surface at the top level. 
However, why `ProjectCostSurfaceModule` instead of `CostSurface` since we mentioned it to be a first class citizen?